.github/assets/docker_compose.env

```diff
@@ -1,3 +1,4 @@
 COMPOSE_PROFILE=_test:_test2
 DEV_SOURCE_PATH=pulpcore:pulp_file
-COMPOSE_BINARY=docker
\ No newline at end of file
+COMPOSE_BINARY=docker
+SECOND_SERVICE_PORT=8002
```

Comment:
What is this for, the "second one started"?

Reply:
That's just for CI. I created a set of profiles for CI, and one of them runs a web server to verify that other services can be added to the system: https://github.com/pulp/oci_env/pull/31/files#diff-50bc831b758ad43f28642443148d9d1f6fc62cce155af3000c4f16d0dbdae959

Originally it just listened to port 8002 by default, but you can't have two containers on the same port, so I had to add a configuration for that.